### PR TITLE
feat: implement block list page

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: web/src/pages/BlockList/index.css.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  display: flex;
+  flex-direction: row;
+
+  th {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 240px;
+    height: ${(props: { width: number }) => (78 * props.width) / 1920}px;
```
Comment (wuyuyue) [inline web/src/pages/BlockList/index.css.tsx]: recommend to keep the height here the original 78; it was not an image fullscreen

Comment (duanyytop) [reply web/src/pages/BlockList/index.css.tsx]: OK

File: web/src/pages/BlockList/index.css.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+export const ContentTable = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 58px;
+`
+
+export const TableTitleRow = styled.tr`
+  background: rgb(75, 188, 142);
+  display: flex;
```
Comment (wuyuyue) [inline web/src/pages/BlockList/index.css.tsx]: tr tag is default as row for its td-child-elements, why keep display: flex, flex-direction: row,  here?

Comment (duanyytop) [reply web/src/pages/BlockList/index.css.tsx]: Without flex, it will show vertically

Comment (wuyuyue) [reply web/src/pages/BlockList/index.css.tsx]: it is a weird layout for table tr, I never use it before. by default, I think tr(table-row) is in a row for a table. maybe here there is no need to use a table, looks like a div if you set the display with flex in tr/td to change the behavior

Comment (duanyytop) [reply web/src/pages/BlockList/index.css.tsx]: OK. I use `div` to implement.

**Pull request merged**